### DISCUSSION TITLE
Replace the PF description by "300 or quit." in osu!mania

### DISCUSSION THREAD

**Discussion by Anax2i** [ppy/osu#29856] [Ideas] [answered]
A small proposal for mania players who are playing osu! in lazer would be to simply change the description mod of **Perfect** to "300 or quit." or something similar. This is to avoid any confusion due to the fact that a SS would no longer make you fail, unless you do something other than a `PERFECT` or a `GREAT`.

![osu_2024-09-12_20-32-50](https://github.com/user-attachments/assets/7bbcba74-692d-480c-b9e3-694ecf39bb55)

Source: https://osu.ppy.sh/community/forums/topics/1976312

**Comment by bdach** [comment, marked as answer]
People want definition of SS to change to what it was on stable (https://github.com/ppy/osu/pull/29640) so this would be counterproductive.
